### PR TITLE
Initial Base Preservation & Devastation Evoker Modules

### PR DESCRIPTION
**Pull request by tyndivelspaz**

### CODE REVIEW

File: src/analysis/retail/evoker/shared/modules/core/LivingFlame.tsx [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+  statistic() {
+    return (
+      <Statistic
+        size="flexible"
+        position={STATISTIC_ORDER.CORE(1)}
+        category={STATISTIC_CATEGORY.TALENTS}
+      >
+        <BoringSpellValueText spellId={SPELLS.LIVING_FLAME_DAMAGE.id}>
+          <ItemPercentHealingDone amount={this.livingFlameHealing} />
```
Comment (emallson) [inline src/analysis/retail/evoker/shared/modules/core/LivingFlame.tsx]: is this supposed to be `ItemPercentHealingDone`? or should it be `ItemHealingDone`?

File: src/parser/core/CombatLogParser.tsx [hunk 2]
```diff
@@ -226,7 +226,7 @@ class CombatLogParser {
     darkmoonDeckVoracity: DarkmoonDeckVoracity,
 
     // Shadowlands
-    effusiveAnimaAccelerator: EffusiveAnimaAccelerator,
+    // effusiveAnimaAccelerator: EffusiveAnimaAccelerator,
```
Comment (emallson) [inline src/parser/core/CombatLogParser.tsx]:
```suggestion
```
deleto

File: src/parser/core/CombatLogParser.tsx [hunk 3]
```diff
@@ -13,7 +13,7 @@ import {
   MappedEvent,
 } from 'parser/core/Events';
 import ModuleError from 'parser/core/ModuleError';
-import EffusiveAnimaAccelerator from 'parser/shadowlands/modules/covenants/kyrian/EffusiveAnimaAccelerator';
+// import EffusiveAnimaAccelerator from 'parser/shadowlands/modules/covenants/kyrian/EffusiveAnimaAccelerator';
```
Comment (emallson) [inline src/parser/core/CombatLogParser.tsx]:
```suggestion
```

**Review by emallson — COMMENTED** [review]
one question but otherwise looks fine

**Comment by tyndivelspaz** [comment]
Done!

**Review by emallson — APPROVED** [review]

**Pull request merged**